pyproject.toml

```diff
@@ -17,7 +17,37 @@ black = "^23.11.0"
 flake8 = { version = "^6.1.0", python = ">=3.8.1,<4" }
 pre-commit = "^3.5.0"
 mypy = "^1.7.0"
+pyright = "^1.1.352"
+ruff = "^0.3.0"
 
 [build-system]
 requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.pyright]
+include = ["whylogs_container", "integ", "tests"]
+typeCheckingMode = "strict"
+
+reportMissingTypeStubs = false
+reportMissingParameterType = false
+reportMissingTypeArgumet = false
+
+[tool.ruff]
+line-length = 140
+indent-width = 4
+include = ["./whylogs_container/**/*.py", "./tests/**/*.py", "./integ/**/*.py"]
```

Comment:
?

Reply:
This is how ruff is configured

Reply:
the `whylogs_container` folders were what I was questioning there, you fixed it now.